products/coldfusion.md

```diff
@@ -18,32 +18,49 @@ releases:
     releaseDate: 2023-05-17
     eol: 2028-05-16
     extendedSupport: 2029-05-16
-    link: null # does not exist yet, will probably become valid after the first update
+    latestReleaseDate: 2023-10-05
+    latest: "ColdFusion (2023 Release) Update 5"
+    link: https://coldfusion.adobe.com/2023/05/coldfusion2023-release/
```

Comment:
```suggestion
```

All links except the first one is already generated by the `changelogTemplate`. So all `link` attributes must be removed (included the first one now that https://helpx.adobe.com/coldfusion/kb/coldfusion-2023-updates.html is available).

Reply:
Merci! I removed all link attributes except the first one (which I left as null).  Can you confirm if this was what you were thinking?

Reply:
Even the first one can be removed. The URL did not exist at the time, but now it is.